File: CSES/Problem_Set/Repetitions.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
    string s;
    cin >> s;
    int ans = 0;
    int temp = 0;
    char l = 'A';
    for (char d:s)
    {
        //cout << d << endl;
        if (d==l)
        {
            temp += 1;
            ans = max(ans, temp);
        }
        else
        {
            l = d;
            temp = 1;
        }
    }
    cout << ans;
}

File: CSES/Problem_Set/Repetitions.py
dna = str(input())

start = dna[0]
count = 1
ans = 1

for i in dna[1:]:
    if i == start:
        count += 1
    else:
        start = i
        ans = max(ans, count)
        count = 1
print(max(ans,count))